website/docs/docs/guides.md

```diff
@@ -0,0 +1,8 @@
+---
+sidebar_position: 6
+title: Guides
+---
+
+Here you can find some guides or recipes on using the Sandpack in different situations. These examples are basic implementations, and you need to consider improving for production, but you can find more detailed guides in the [advanced usage](/advanced-usage/provider) section.
+
+- Prettier: https://codesandbox.io/s/sandpack-prettier-1po91?file=/src/App.js
```

Comment:
Super nice idea to have more guides and examples!!

Maybe we can add just a very brief sentence instead of one word? Something like
```suggestion
- Integrate Prettier for code formatting: [CodeSandbox example](https://codesandbox.io/s/sandpack-prettier-1po91?file=/src/App.js)
```